core/kazoo_perf/src/kazoo_perf_maintenance.erl

```diff
@@ -0,0 +1,119 @@
+
+collect() ->
+    {Mega, Secs, _Micro} = os:timestamp(),
+    put(timestamp, Mega * 1000000 + Secs),
+    Metrics = [memory_statistics
+              ,number_of_processes
+              ,context_switches
+              ,garbage_collection
+              ,io
+              ,scheduler_reductions
+              ,processes_in_run_queue_of_each_schedulers
+              ,ets_tables_sizes
+              ],
+    Malt = [1, {processes, length(Metrics)}],
+    plists:map(fun (Metric) -> {Metric, measure(Metric)} end, Metrics, Malt).
+
+scheme(Account, Cluster, Zone) ->
+    [Service, Hostname0] = binary:split(kz_term:to_binary(node()), <<$@>>),
+    Hostname = binary:replace(Hostname0, <<$.>>, <<"::">>, [global]),
+    kz_util:iolist_join($., [Account, Cluster, Zone, Hostname, Service]).
+
+print_metric(Scheme, Key, Value) ->
+    io:format("~s.~s ~B ~B\n", [Scheme, Key, Value, get(timestamp)]).
+
+graphite(Scheme, memory_statistics, MemoryMetrics) ->
+    [print_metric(Scheme, "memory_" ++ atom_to_list(Key), Value)
+     || {Key, Value} <- MemoryMetrics
+    ];
+graphite(Scheme, number_of_processes, LiveProcesses) ->
+    print_metric(Scheme, processes, LiveProcesses);
+graphite(Scheme, context_switches, {Switches, 0}) ->
+    print_metric(Scheme, context_switches, Switches);
+graphite(Scheme, garbage_collection, {NumberofGCs, WordsReclaimed, 0}) ->
+    print_metric(Scheme, number_of_gcs, NumberofGCs),
+    print_metric(Scheme, words_reclaimed, WordsReclaimed);
+graphite(Scheme, io, {{input, Input}, {output, Output}}) ->
+    print_metric(Scheme, input_io_bytes, Input),
+    print_metric(Scheme, output_io_bytes, Output);
+graphite(Scheme, scheduler_reductions, {TotalReductions, _ReductionsSinceLastCall}) ->
+    print_metric(Scheme, reductions, TotalReductions);
+graphite(Scheme, processes_in_run_queue_of_each_schedulers, RunQueue) ->
+    print_metric(Scheme, run_queue, RunQueue);
+graphite(Scheme, ets_tables_sizes, Tabs) ->
+    [print_metric(Scheme, "tab_" ++ kz_term:to_list(Tab), Size)
+     || {Tab, Size} <- Tabs,
+        Size =/= 0
+    ].
+
+to_props(memory_statistics, MemoryMetrics) ->
+    [{"memory_" ++ atom_to_list(Key), Value}
+     || {Key, Value} <- MemoryMetrics
+    ];
+to_props(number_of_processes, LiveProcesses) ->
+    [{processes, LiveProcesses}
+    ];
+to_props(context_switches, {Switches, 0}) ->
+    [{context_switches, Switches}
+    ];
+to_props(garbage_collection, {NumberofGCs, WordsReclaimed, 0}) ->
+    [{number_of_gcs, NumberofGCs}
+    ,{words_reclaimed, WordsReclaimed}
+    ];
+to_props(io, {{input, Input}, {output, Output}}) ->
+    [{input_io_bytes, Input}
+    ,{output_io_bytes, Output}
+    ];
+to_props(scheduler_reductions, {TotalReductions, _ReductionsSinceLastCall}) ->
+    [{reductions, TotalReductions}
+    ];
+to_props(processes_in_run_queue_of_each_schedulers, RunQueue) ->
+    [{run_queue, RunQueue}
+    ];
+to_props(ets_tables_sizes, Tabs) ->
+    [{"tab_" ++ kz_term:to_list(Tab), Size}
```

Comment:
@swysor says that graphite needs it but yes I wi;; remove it from the json output.